### PR TITLE
Added final Spanish translations

### PR DESCRIPTION
**Pull request by montogeek**

### CODE REVIEW

**Comment by montogeek** [comment]
CLA signed!

File: src/_langs/es/_betterbook.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 toc:
-  - title: "Web <strong>Fundamentals</strong>"
+  - title: "Aspectos básicos de la <strong>Web</strong>"
```
Comment (PaulKinlan) [inline src/_langs/es/_betterbook.yaml]: Is there a better word here?  It is not really the basics, it is the core principles.

Comment (PaulKinlan) [inline src/_langs/es/_betterbook.yaml]: Also, Web Fundamentals is a brand name :)

**Comment by PaulKinlan** [comment]
@montogeek thank you very much.  It should be live soon.

**Pull request merged**